azurerm/resource_arm_securitycenter_contact_test.go

```diff
@@ -0,0 +1,109 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMSecurityCenterContact_basic(t *testing.T) {
+	resourceName := "azurerm_securitycenter_contact.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMSecurityCenterContact_template("email1@example.com", "+1-555-555-5555", true, true),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMSecurityCenterContactExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "email", "email1@example.com"),
+					resource.TestCheckResourceAttr(resourceName, "phone", "+1-555-555-5555"),
+					resource.TestCheckResourceAttr(resourceName, "alert_notifications", "true"),
+					resource.TestCheckResourceAttr(resourceName, "alerts_to_admins", "true"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccAzureRMSecurityCenterContact_update(t *testing.T) {
+	resourceName := "azurerm_securitycenter_contact.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMSecurityCenterContact_template("email1@example.com", "+1-555-555-5555", true, true),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMSecurityCenterContactExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "email", "email1@example.com"),
+					resource.TestCheckResourceAttr(resourceName, "phone", "+1-555-555-5555"),
+					resource.TestCheckResourceAttr(resourceName, "alert_notifications", "true"),
+					resource.TestCheckResourceAttr(resourceName, "alerts_to_admins", "true"),
+				),
+			},
+			{
+				Config: testAccAzureRMSecurityCenterContact_template("email2@example.com", "+1-555-678-6789", false, false),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMSecurityCenterContactExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "email", "email2@example.com"),
+					resource.TestCheckResourceAttr(resourceName, "phone", "+1-555-678-6789"),
```

Comment:
could we make this an international number to ensure that works too? `+44 1234 567890` should work as an example